spec/services/offer_validations_spec.rb

```diff
@@ -138,6 +138,16 @@
         end
       end
 
+      context 'when a provider attemps to make an offer with other application as conditions not met' do
+        let!(:application_form) { create(:application_form, application_choices: [application_choice, other_application_choice]) }
+        let(:application_choice) { build(:application_choice, :offered, current_course_option: course_option) }
+        let!(:other_application_choice) { build(:application_choice, :conditions_not_met) }
+
+        it 'be valid' do
```

Comment:
```suggestion
        it 'is valid' do
```

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/9141/commits/f78a6bf173dff8eb57aefcd472371c6ad5589a45